### ISSUE TITLE
[v0.12.2] Error when trying to create continuous query

### ISSUE THREAD

**Issue by SimonSimCity** [apache/iotdb#4325]
**Describe the bug**

Today I wanted to create a continuous query, so I gave it a run. Already when running the first query I got from the documentation I got an error - and whatever I wrote, I got the same error:

```
Msg: 401: Error occurred while parsing SQL to physical plan: line 1:7 no viable alternative at input 'CREATE CONTINUOUS'
```

**To Reproduce**
1. Open a client to the iot-db
2. Run a query like `CREATE CONTINUOUS QUERY test BEGIN SELECT foo INTO test FROM root GROUP BY time(10s) END`

**Expected behavior**

Create a continuous query of some kind ...

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser n/a
 - Version 0.12.2

**Additional context**

n/a

**Comment by SimonSimCity** [comment]
I guess almost this will also come with 0.13? 😄

**Comment by SteveYurongSu** [comment]
> I guess almost this will also come with 0.13? 😄
> 
> At least according to the git-history (comparing the merge of the feature - [5a862ee](https://github.com/apache/iotdb/commit/5a862eef042ce365da1bcf5b2bb4999a6b897357) - to the tag of `0.12.2` - https://github.com/apache/iotdb/blob/v0.12.2/antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4) it's not in.

@SimonSimCity Yes, CQ will come with 0.13.x. You can download our repo, checkout to the master branch, build and have a try :D
